lighthouse/classes/event_properties/validations.py

```diff
@@ -1,10 +1,25 @@
-from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract  # type: ignore
+from typing import Protocol, Dict, Optional
+
+
+class EventPropertyProtocol(Protocol):
```

Comment:
I think we should add this to a typing namespace perhaps?

Reply:
I'll move it to lighthouse.types

Reply:
Done